### PR TITLE
EPMFARMATS-12781

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const yargs = require('yargs').argv;
```
Comment (SiarheiRylach) [inline config/config.js:1]: As far as I see both of your specs fail by now, when I run it

File: po/AbstractElement.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        return elementText;
+    };
+
+    sendKeys(keys) {
+        logger.debug(`Sending "${keys}" to "${this.name}"`);
+        return this.elementFinder.sendKeys(keys);
+    }
+
+    isPresent() {
+        logger.debug(`Comparing "${this.name}"`);
```
[the remark this answers is not in the document]
Comment (overdd) [reply po/AbstractElement.js:40]: Fixed

File: testresults/junitresults-firefox.Deltaairlinesbookingsample.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
[the remark this answers is not in the document]
Comment (overdd) [reply testresults/junitresults-firefox.Deltaairlinesbookingsample.xml:1]: fixed

File: README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+EPMFARMATS-12781 test run example:
+protractor config/config.js --browser="firefox" --instances=2 --suite smoke
```
[the remark this answers is not in the document]
Comment (overdd) [reply README.md:2]: Fixed

File: config/config.js (hunk 1 above)
Comment (overdd) [reply config/config.js:1]: Fixed

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**